File: test11/devowel_file.pl
#!/usr/bin/perl -w

@lines = ();

open F, "<", $ARGV[0] or die "$0: $!\n";
while ($line = <F>) {
   chomp $line;
   $line =~ s/[aeiouAEIOU]//g;
   push @lines, $line;
}
close F;

open F, ">", $ARGV[0] or die "$0: $!\n";
for $line (@lines) {
   print F "$line\n";
}
close F;

File: test11/devowel_file.py
#!/usr/bin/python3

import sys
import re

lines = []
with open(sys.argv[1], 'r') as f:
   for line in f:
      line = re.sub(r'[aeiouAEIOU]', '', line)
      lines.append(line)

with open(sys.argv[1], 'w') as f:
   for line in lines:
      print(line, end='', file=f)
